lib/customer.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'menu'
+
+class Customer
+  include Menu
+
+  attr_reader :order, :menu, :total_price
+
+  def initialize(menu)
+    @menu = menu
+    @order = {}
+    @total_price = 0
+  end
+
+  def add!(dishes, total_price)
+    @order = @order.merge(dishes)
+    @total_price += total_price
+
+    total_value = 0
+    dishes.each do |dish, quantity|
+      total_value += quantity * dishes_list[dish]
+    end
+  end
+end
```

Comment:
Single Responsibility Principle!
How many things does the customer do/know? He knows about the business logic of the order, calculates something, has a menu that it's not being used ... He should only be able to place an order, without knowing how this is implemented.